miden-tx/src/host/account_delta.rs

```diff
@@ -75,17 +243,15 @@ impl AccountVaultDeltaTracker {
             Asset::NonFungible(asset) => {
                 update_asset_delta(&mut self.non_fungible_assets, asset.vault_key().into(), -1)
             },
-        };
-
-        Ok(())
+        }
     }
 
     // CONVERSIONS
     // --------------------------------------------------------------------------------------------
 
     /// Consumes this delta tracker and returns the [AccountVaultDelta] that represents the changes
```

Comment:
"the" instead of "this"? 

Or could be "this" as in OOP class.

Reply:
I changed it to `Consumes self and ...` as I think this is better.